packages/jest-phabricator/src/index.ts

```diff
@@ -32,7 +34,10 @@ function summarize(coverageMap: CoverageMap): CoverageMap {
       }
     }
 
-    summaries[file] = covered.join('');
+    summaries[file] = {
```

Comment:
This is a breaking change to `jest-phabricator`, necessary to support the JSON lines format.

Technically, before it was breaking the contract provided by TypeScript by just returning an `any`... I've typed it correctly now.